wgengine/router/router_linux.go

```diff
@@ -129,6 +130,12 @@ func chooseFireWallMode(logf logger.Logf, det tableDetector) linuxfw.FirewallMod
 func newNetfilterRunner(logf logger.Logf) (netfilterRunner, error) {
 	tableDetector := &linuxFWDetector{}
 	mode := chooseFireWallMode(logf, tableDetector)
+	if mode == linuxfw.FirewallModeIPTables {
+		hostinfo.SetFirewallMode("iptables")
+	} else {
+		hostinfo.SetFirewallMode("")
```

Comment:
let's remove the empty-means-nftables bit. If we want to save some bytes, we can up "ipt" and "nft" here.

Reply:
The more interesting bit for stats is _why_ we chose whatever path. Maybe we should move these SetFirewallMode calls to chooseFireWallMode before each return statement.

e.g.

     hostinfo.SetFirewallMode("nft-inuse")

Reply:
I don't think we are interested in the reason we choose ipt or nft. The reasons is we are always prioritizing nft. In what circumstances would we usually want that bit? Like what's the background we might use that stat?

Reply:
This would also be hard for user or support team to understand what's going on since they don't necessarily understand our prioritization? I don't know but It just seems complicated comparing to other fields in NetInfo

Reply:
Support team can look at the prefix if they only want the current state. I'm interesting in learning how we transition this over time and what the state of the world looks like (like: how many people are forcing this? what are we failing on?) Also, support could be interested in knowing why a path was taken. Logs can answer that, but so could this.

Or we could add clientmetrics too.

In any case, at least remove the zero value meaning nftables. That's too confusing as implemented with showing the empty string in some log statements to mean nftables when the empty string might also mean neither.

Reply:
Updated, thoughts on the latest?